uk.co.compucorp.civicrm.hrcore/tests/phpunit/CRM/HRCore/Menu/MenuBuilderTest.php

```diff
@@ -0,0 +1,169 @@
+<?php
+
+use CRM_HRCore_Menu_Config as MenuConfig;
+use CRM_HRCore_Menu_MenuBuilder as MenuBuilder;
+use CRM_HRCore_Menu_Item as MenuItem;
+
+/**
+ * @group headless
+ */
+class CRM_HRCore_Menu_MenuBuilderTest extends CRM_HRCore_Test_BaseHeadlessTest {
+
+  public function testMenuConfigItemKeyIsSetAsMenuItemLabel() {
+    $menuConfigItem = [
+      'Staff' => []
+    ];
+
+    $menuConfig = $this->prophesize(MenuConfig::class);
+    $menuConfig->getItems()->willReturn($menuConfigItem);
+    $menuBuilder = new MenuBuilder();
+
+    $menuObject = $menuBuilder->getMenuItems($menuConfig->reveal());
+    $children = $menuObject->getChildren();
+    $this->assertCount(1, $children);
+    //'Staff' which is the Item key is set as the MenuItem label
+    $this->assertEquals('Staff', $children[0]->getLabel());
+  }
+
+  public function testMenuConfigItemValueWillBeSetAsMenuItemUrlIfItIsAString() {
+    $url = 'civcrm/staff';
+    $menuConfigItem = [
+      'Staff' => $url
+    ];
+
+    $menuConfig = $this->prophesize(MenuConfig::class);
+    $menuConfig->getItems()->willReturn($menuConfigItem);
+    $menuBuilder = new MenuBuilder();
+
+    $menuObject = $menuBuilder->getMenuItems($menuConfig->reveal());
```

Comment:
These 4 lines of code (from 34 to 38) are repeated in all the tests here. I suggest we avoid that by doing this:

- Create a property in this class to store the MenuBuilder instance
- Instantiate the MenuBuilder in the setUp method
- Create a helper method to create a MenuConfig mock that returns a given menu configuration